packages/content-api/api/pages.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    docName: 'pages',
+    browse: {
+        options: [
```

Comment:
Is it necessary to include `options` in this package? It seems like that's creating duplication between this and Ghost that would be necessary to keep the two in sync when anything changes.

I would suggest it's Ghost's responsibility to filter the params rather than trying to filter before passing on to Ghost.

Reply:
Yeah that's fair - will remove :+1: